common/src/test/java/com/sequenceiq/cloudbreak/util/StaticFieldManipulationTestHelper.java

```diff
@@ -11,9 +12,21 @@ private StaticFieldManipulationTestHelper() {
     }
 
     public static void setFinalStatic(Field field, Object newValue) throws Exception {
+        // TODO: rewrite it!!!
+        // It is an ugly and fragile hack and it needs opening Java internal modules with
+        // --add-opens java.base/java.lang=ALL-UNNAMED --add-opens java.base/java.lang.reflect=ALL-UNNAMED --add-opens java.base/java.util.concurrent=ALL-UNNAMED
```

Comment:
please ensure we have this in a ticket too

Reply:
+1 There will indeed be a follow-up ticket for all the hacks here.

Reply:
Story created under [epic](https://jira.cloudera.com/browse/CB-21717)